vscode/src/workspace.ts

```diff
@@ -193,7 +193,7 @@ export class Workspace implements WorkspaceInterface {
     });
 
     // If the gem is not yet installed, install it
-    if (!stdout.includes("ruby-lsp")) {
+    if (!/\bruby-lsp\b/.exec(stdout)) {
```

Comment:
🤔 is this right? `_` would be considered part of the word, but `-` isn't:

```
irb(main):011> "ruby-lsp-rails".match?(/\bruby-lsp\b/)
=> true
```

Reply:
It wasn't indeed. The hyphen also matches as a word boundary. I changed the regex to ensure we match properly.